database/revert/V20170114_2116__createAttendeeRSVP.revert.sql

```diff
@@ -1 +1,5 @@
-DROP TABLE `attendee_rsvps`;
\ No newline at end of file
+DROP TABLE `attendee_rsvps`;
+DELETE FROM `mailing_lists` WHERE `name`='attendees';
```

Comment:
Why are you deleting attendees? I'm pretty sure we're already using this list in production